pycbc/workflow/jobsetup.py

```diff
@@ -960,6 +1114,42 @@ def create_node(self, job_segment, input_file):
                                  tags=self.tags, store_file=self.retain_files)
         return node
 
+class PycbcSplitInspinjExecutable(Executable):
+    """
+    The class responsible for running the pycbc_split_inspinj executable
+    """
+    current_retention_level = Executable.INTERMEDIATE_PRODUCT
+    def __init__(self, cp, exe_name, num_splits, universe=None, ifo=None,
+                 out_dir=None, tags=[]):
+        super(PycbcSplitInspinjExecutable, self).__init__(cp, exe_name,
+                universe, ifo, out_dir, tags)
+        self.out_dir = out_dir
+        self.num_splits = int(num_splits)
+    def create_node(self, parent):
+        node = Node(self)
+
+        node.add_input_opt('--input-file', parent)
+        
+        if parent.name.endswith("gz"):
+            ext = ".xml.gz"
+        else:
+            ext = ".xml"
+
+        out_files = FileList([])
+        for i in range(self.num_splits):
+            curr_tag = 'split%d' % i
+            # FIXME: What should the tags actually be? The job.tags values are
+            #        currently ignored.
```

Comment:
I have done what I thnk you're suggesting in https://github.com/a-r-williamson/pycbc/commit/3a161f4bafa52bd61e1f93fc9911e1122b0371a4. Can you confirm this is what you want?